spec/spec_helper.rb

```diff
@@ -0,0 +1,112 @@
+ENV["RAILS_ENV"] ||= "test"
```

Comment:
I see you add this file in 73b3e68ac3413667ae69fb67c2bf0695325ca4ae and then immediately remove it in 0dc7c2aff023e2766ac6bbc36e0b16b3bc7c9c41. Can you rebase your branch to strip out these unnecessary commits?

Reply:
👍